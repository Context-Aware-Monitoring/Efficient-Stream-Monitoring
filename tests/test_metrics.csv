now,cpu.user,mem.used,load.cpucore,load.min1,load.min5,load.min15
2019-11-25 15:58:58 CEST,15.0,11300000000,8,1.99,1.93,2.11
2019-11-25 15:58:58 CEST,14.5,11350000000,8,2.00,1.94,2.12
2019-11-25 15:58:58 CEST,15.5,11400000000,8,2.01,1.95,2.13
2019-11-25 15:58:59 CEST,16.0,11200000000,8,1.99,1.94,2.13
2019-11-25 15:58:59 CEST,16.5,11250000000,8,2.00,1.93,2.12
2019-11-25 15:58:59 CEST,15.5,11300000000,8,2.01,1.95,2.11
2019-11-25 16:59:00 CEST,15.5,11300000000,8,2.01,1.95,2.11