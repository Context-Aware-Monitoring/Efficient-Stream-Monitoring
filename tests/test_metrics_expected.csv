now,cpu.user,mem.used,load.cpucore,load.min1,load.min5,load.min15
2019-11-25 15:58:58 CEST,15.0,11350000000.0,8.0,2.0,1.94,2.12
2019-11-25 15:58:59 CEST,16.0,11250000000.0,8.0,2.0,1.94,2.12
2019-11-25 16:59:00 CEST,15.5,11300000000.0,8.0,2.01,1.95,2.11
